examples/graphene_apps/python_worker/avalon_worker/work_order_processor.py

```diff
@@ -344,57 +338,37 @@ def _create_work_order_response(self, input_json, out_data,
 
     def _get_mrenclave(self):
         """
-        Get mrenclave value of Graphene-SGX worker enclave.
+        Get mrenclave value of worker enclave.
 
         Returns :
-            mrenclave value in case of Graphene-SGX worker as hex string.
+            mrenclave value of worker enclave as hex string.
             If Intel SGX environment is not present returns empty string.
         """
-        try:
-            # Read target info
-            target_info_file = \
-                WorkOrderProcessor.GRAPHENE_SGX_TARGET_INFO_FILE
-            with open(target_info_file, 'rb') as fd:
-                data = fd.read()
-                mrencalve = data[:32].hex()
-        except Exception:
-            logger.warn("Graphene-SGX Environment not setup."
-                        "Return empty mrencalve string.")
-            mrencalve = ""
-        return mrencalve
+        mrenclave = self.sgx_attestation.get_mrenclave()
+        return mrenclave
 
 # -------------------------------------------------------------------------
 
     def _get_quote(self):
         """
-        Get quote of Graphene-SGX worker enclave.
+        Get quote of worker enclave.
 
         Returns :
-            quote value in case of Graphene-SGX worker as hex string.
+            quote value of worker as base64 encoded string.
             If Intel SGX environment is not present returns empty string.
         """
-        try:
-            # Write user report data
-            user_report_data_file = \
-                WorkOrderProcessor.GRAPHENE_SGX_USER_REPORT_DATA_FILE
-            with open(user_report_data_file, 'wb') as file:
-                # First 32 bytes of report data has SHA256 hash of worker's
-                # public signing key. Next 32 bytes is filled with Zero.
-                hash_pub_key = sha256(self.worker_public_sign_key).digest()
-                user_data = hash_pub_key + bytearray(32)
-                file.write(user_data)
-                logger.debug("user_data hex = {}".format(user_data.hex()))
```

Comment:
This log was good at debug level. Not divulging anything private.

Reply:
Log is still there. Moved to a different file though